packages/svelte/src/compiler/phases/2-analyze/css/Stylesheet.js

```diff
@@ -48,7 +48,7 @@ function escape_comment_close(node, code) {
 	}
 }
 
-class Rule {
+export class Rule {
```

Comment:
doesn't look like this imported by another module

```suggestion
class Rule {
```